kuma/search/filters.py

```diff
@@ -16,6 +20,17 @@ def get_filters(getter_func):
     return filters.keys()
 
 
+class QueryStringValidationMixin(object):
+    """
+    A mixin to validate query string parameters.
+    """
+    def validate_query_params(self, request):
```

Comment:
This seems be called three times for each request, which seems wasteful. Couldn't we run this once per request in the view and set a view parameter? The filter methods do get the view instance passed along after all.